skimage/io/collection.py

```diff
@@ -100,19 +100,18 @@ class ImageCollection:
 
     Parameters
     ----------
-    load_pattern : str or list of str
-        Pattern string or list of strings to load. The filename path can be
-        absolute or relative.
+    load_pattern : str or sequence
+        A glob-like pattern or sequence of patterns to load files. The pattern can be
+        absolute or relative. If a sequence of objects other than strings is passed,
```

Comment:
I think the absolute / relative is an intuitive concept with paths, but confusing in the context of this docstring, so perhaps leave out.